test/format/format.sh

```diff
@@ -437,13 +442,22 @@ format()
 	fi
 
 	cmd="$format_binary -c "$config" -h "$dir" -1 $args quiet=1"
-	verbose "$name: $cmd"
+	echo "$name: $cmd"
 
 	# Disassociate the command from the shell script so we can exit and let the command
 	# continue to run.
 	# Run format in its own session so child processes are in their own process gorups
 	# and we can individually terminate (and clean up) running jobs and their children.
-	nohup setsid $cmd > $log 2>&1 &
+	eval $env_var setsid $cmd > $log 2>&1 &
```

Comment:
What environment variable are you planning to pass to the `setsid` command?

Reply:
It's `SEGFAULT_SIGNALS=all`. Passing the env variable after setsid would fail as below:

> setsid: failed to execute SEGFAULT_SIGNALS=all: No such file or directory